### PR TITLE
C#/Unity SDK Testsuite

### PR DESCRIPTION
**Pull request by jdetter**
## Description

We're moving the Unity code from the C#/Unity testsuite here to prevent duplication of the circle game project.

This testsuite works against this specific revision of SpacetimeDB: `0935b7346b825b8cbb9f36d9ed256136b73b5f0b`

## Testing

Testsuite passes:

image

If you want to double-check my work, you can: 

```
cd SpacetimeDB
git checkout 0935b7346b825b8cbb9f36d9ed256136b73b5f0b
cargo install --path ./crates/cli
cd ../SpacetimeDBCircleGame/server
bash ./generate.sh
bash ./publish.sh
```

## Merging notes

You'll have to wait for 1.0.0 to release before merging this, and make sure you update the package before merging. You just have to hit the update button in the package manager and commit the package.lock change

image

### CODE REVIEW

File: client/Assets/Scripts/GameManager.cs [hunk 1]
```diff
@@ -43,6 +48,7 @@ private void Start()
     {
         instance = this;
         Application.targetFrameRate = 60;
+        PlayerPrefs.DeleteAll();
```
Comment (jdetter) [inline client/Assets/Scripts/GameManager.cs:51]:
```suggestion
```

File: client/.gitignore
Comment (cloutiertyler) [inline client/.gitignore:1]: Amen.

File: client/Packages/manifest.json [hunk 2]
```diff
@@ -1,13 +1,14 @@
 {
   "dependencies": {
-    "com.clockworklabs.spacetimedbsdk": "https://github.com/clockworklabs/com.clockworklabs.spacetimedbsdk.git",
+    "com.clockworklabs.spacetimedbsdk": "file:/Users/boppy/clockwork/com.clockworklabs.spacetimedbsdk",
```
Comment (cloutiertyler) [inline client/Packages/manifest.json:3]: I'm assuming you didn't mean to have this?

File: server/Cargo.toml [hunk 3]
```diff
@@ -9,6 +9,6 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
-spacetimedb = { version = "0.12.0" }
+spacetimedb = { path = "../../SpacetimeDB/crates/bindings" }
```
Comment (cloutiertyler) [inline server/Cargo.toml:12]: Same here.

**Review by cloutiertyler — CHANGES_REQUESTED** [review]
Left two comments.

**Review by cloutiertyler — APPROVED** [review]
This now LGTM

**Pull request merged**